dashboard/src/components/ChartView/ChartDeployButton.tsx

```diff
@@ -31,7 +33,7 @@ class ChartDeployButton extends React.Component<IChartDeployButtonProps, IChartD
         {this.state.clicked && (
           <Redirect
             push={true}
-            to={`/ns/${namespace}/apps/new/${repoName}/${chartName}/versions/${versionStr}`}
+            to={`/ns/${namespace}/apps/${newSegment}/${repoName}/${chartName}/versions/${versionStr}`}
```

Comment:
same here about the query param `global`

Reply:
Yeah, for similar reasons, using a query param would indicate an optional, non-hierarchical, piece of data - this URI path unambiguously identifies the hierarchy of the chart (ie. a chart in the global repository called `${repoName}` with the specific `${chartName}` and `${versionStr}`.